converters/fromZigbee.js

```diff
@@ -5777,6 +5777,14 @@ const converters = {
             return result;
         },
     },
+    rc_110_level_to_scene: {
```

Comment:
Why did you choose to add a custom converter instead of using `fz.command_move_to_level`?

Reply:
The scenes buttons are reported as specific values and transition times which didn’t really make much sense from a user perspective (a pressed button will always be this value) as the buttons on the device are labelled as scenes, the returned brightness level isn’t really useful

Reply:
thanks!